windows/CodePushUnknownException.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
```

Comment:
Its any unexpected exception thrown can be annotated with our error message and that it is indeed coming from CodePush.